build/images/perftool/Dockerfile.windows

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/windows/servercore:1809

```

Comment:
Could you consider adding a workflow like this one so that the Docker image can easily be built and pushed to the registry when needed: https://github.com/antrea-io/antrea/blob/main/.github/workflows/docker_update_ethtool.yml

Reply:
Added.